tools/image_processing/yolo/doclayoutyolo.xml

```diff
@@ -0,0 +1,120 @@
+<tool id="doclayout_yolo" name="DocLayout-YOLO: Enhancing Document Layout Analysis" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>through Diverse Synthetic Data and Global-to-Local Adaptive Perception</description>
+    <macros>
+        <token name="@TOOL_VERSION@">0.0.4.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@PROFILE@">23.2</token>
+    </macros>
+    <creator>
+        <organization name="European Galaxy Team" url="https://galaxyproject.org/eu/"/>
+        <person givenName="Anup" familyName="Kumar" email="kumara@informatik.uni-freiburg.de"/>
+    </creator>
+    <requirements>
+        <container type="docker">quay.io/galaxy/doclayout-yolo:@TOOL_VERSION@</container>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+    python '$__tool_directory__/main.py'
+            --yolo_model '$input_yolo_model'
+            --input_image '$input_image'
+            --input_image_ext '$input_image.ext'
+            --input_confidence '$input_confidence'
+            --input_image_size '$input_image_size'
+            --output_image '$output_image'
+            --output_geojson '$output_segmentation_coordinates'
+]]>
+    </command>
+    <inputs>
+	    <param name="input_yolo_model" type="data" format="zip" label="Yolo model" help="Please upload a Yolo model."/>
+        <param name="input_image" type="data" format="tiff,jpg,png" label="Input image" help="Please provide an input image for the analysis."/>
+        <param name="input_confidence" type="float" label="Confidence" value="0.5" min="0.0" max="1.0" help="Set confidence threshold between 0.0 and 1.0 drawing bounding boxes"/>
+        <param name="input_image_size" type="integer" label="Image size" value="1024" min="1" max="1500" help="Set input image size for image resize by Doclayout Yolo model"/>
+    </inputs>
+    <outputs>
+        <data format_source="input_image" name="output_image" label="Segmented image"></data>
+        <data format="geojson" name="output_segmentation_coordinates" label="Segmented coordinates"></data>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_yolo_model" value="input_yolo_model.zip" location="https://huggingface.co/juliozhao/DocLayout-YOLO-DocLayNet-Docsynth300K_pretrained/resolve/main/doclayout_yolo_doclaynet_imgsz1120_docsynth_pretrain.pt?download=true"/>
+            <param name="input_image" value="input_image_png.png"/>
+            <param name="input_confidence" value="0.5"/>
+            <param name="input_image_size" value="1024"/>
+            <output name="output_image" ftype="png">
+                <assert_contents>
+                    <has_size size="920950" delta="100" />
```

Comment:
Please also try to use the new image assertions. (even if the linting will fail then)

Reply:
I added a few of these asserts for output images

```
<has_image_channels channels="3" />
<has_image_height height="2721" />
<has_image_width width="1700" />
```

But, the tool tests fail for TIFF output image with:

`<COMPRESSION.LZW: 5> requires the 'imagecodecs' package`